### ISSUE TITLE
Text Summarisation template when deployed on heroku stuck at loading screen

### ISSUE THREAD

**Issue by SupreethRao99** [heartexlabs/label-studio#2769]
**Describe the bug**
LabelStudio is stuck at the loading screen after creating a new text summarisation project. I used the one-click deploy to heroku to run label studio. After creating a project and uploading the data (4MB is size) and selecting the text summarisation template, the UI viewer does not render. After creating the project the screen is stuck at the loading screen.

**To Reproduce**
- Click on one click deploy to heroku on readme page
- create project and upload below file as data
- select text summarisation as template
[test.csv](https://github.com/heartexlabs/label-studio/files/9231013/test.csv)

**Expected behavior**
Label studio should start up and allow the user annotate data

**Screenshots**
![Screenshot 2022-08-01 at 10 29 19 AM](https://user-images.githubusercontent.com/55043035/182075566-81e2e9c6-2b14-40c4-8f06-c2f7f763e1bc.png)

**Environment (please complete the following information):**
 - OS: Heroku 
 - Label Studio Version [e.g. 0.8.0]

**Additional context**
Is there an issue in the way the data is formatted?

**Comment by makseq** [comment]
@SupreethRao99 It's recommended to store data outside of LS and provide links only for LS, e.g.:

```
{
 "text": "http://example.com/1.txt"
}
```

with label config:

```
...
<Text valueType="url" value="text"> 
...
```

**Comment by SupreethRao99** [comment]
thank you @makseq, I'll try that, but I was running into the same issue while running Label Studio locally as a docker container too. in that case I was storing the data in the directory created by the container

**Comment by SupreethRao99** [comment]
@makseq, im facing the same issue when I get data from an S3 bucket

**Comment by makseq** [comment]
@SupreethRao99 are you using Safari?
What is one text size for one task?

**Comment by SupreethRao99** [comment]
> @SupreethRao99 are you using Safari?
> 
> What is one text size for one task? 

Hello, yes , I'm using the latest version of safari, each document that needs to be summarised is about 200-300 words in length

**Comment by makseq** [comment]
What is your label studio version? You need to use 1.5.0post0.

**Comment by SupreethRao99** [comment]
The version in the docker container is 1.15.0, I will try running the latest version. Could you build a container with the latest version , thank you !

**Comment by makseq** [comment]
as I remember there should be `:1.5.0post0` docker image. Or you can try `:latest`

**Comment by SupreethRao99** [comment]
Thank you, I will try that. Also, label studio running on Heroku is no longer stuck at the loading screen when viewed using Google Chrome

**Comment by makseq** [comment]
Yes, it's well-known issue that was fixed in 1.5.0post0.
